Remotion/Web/ClientScript/Scripts/Utilities.ts

```diff
@@ -363,4 +409,15 @@ class WebServiceUtility
         }
       });
   };
+}
+
+class LayoutUtility {
```

Comment:
What about `.outerHeight(true)`? This would be `.offsetHeight` with the margin, right?